src/libraries/Microsoft.Extensions.Configuration.UserSecrets/tests/MsBuildTargetTest.cs

```diff
@@ -58,7 +58,7 @@ public void GeneratesAssemblyAttributeFile(string projectExt, string sourceExt)
                 .GetCustomAttributes<AssemblyMetadataAttribute>()
                 .First(f => f.Key == "MicrosoftNETCoreAppRefPackageVersion")
                 .Value;
-            var target = Path.Combine(_solutionRoot.FullName, "src", "Configuration", "Config.UserSecrets", "src", "build", "netstandard2.0", "Microsoft.Extensions.Configuration.UserSecrets.targets");
+            var target = Path.Combine(_solutionRoot.FullName, "src", "libraries", "Microsoft.Extensions.Configuration.UserSecrets", "src", "build", "netstandard2.0", "Microsoft.Extensions.Configuration.UserSecrets.targets");
```

Comment:
Here's how the test fails for me:

```
 Could not execute because the application was not found or a compatible .NET Core SDK is not installed.
      Possible reasons for this include:
        * You intended to execute a .NET Core program:
            The application 'restore' does not exist.
        * You intended to execute a .NET Core SDK command:
            A compatible installed .NET Core SDK for global.json version [3.1.101] from [C:\CodeHub\runtime\global.json] was not found.
            Install the [3.1.101] .NET Core SDK or update [C:\CodeHub\runtime\global.json] with an installed .NET Core SDK:
              It was not possible to find any installed .NET Core SDKs.
            Install a .NET Core SDK from:
              https://aka.ms/dotnet-download
          Microsoft.Extensions.Configuration.UserSecrets.MsBuildTargetTest.GeneratesAssemblyAttributeFile(projectExt: ".csproj", sourceExt: ".cs") [FAIL]
            Assert.Equal() Failure
            Expected: 0
            Actual:   -2147450735
            Stack Trace:
              C:\CodeHub\runtime\src\libraries\Microsoft.Extensions.Configuration.UserSecrets\tests\MsBuildTargetTest.cs(185,0): at Microsoft.Extensions.Configuration.UserSecrets.MsBuildTargetTest.AssertDotNet(String args)
              C:\CodeHub\runtime\src\libraries\Microsoft.Extensions.Configuration.UserSecrets\tests\MsBuildTargetTest.cs(142,0): at Microsoft.Extensions.Configuration.UserSecrets.MsBuildTargetTest.GeneratesAssemblyAttributeFile(String projectExt, String sourceExt)
            Output:
              Tempdir = C:\CodeHub\runtime\artifacts\bin\Microsoft.Extensions.Configuration.UserSecrets.Tests\netcoreapp5.0-Debug\h4jyyrq2.zis
```

Reply:
I see, we don't actually have an SDK, that makes sense.  We just have host + runtime.  I'd say disable this for now and file an issue to get coverage.